example/lib/src/screens/private_view_page.dart

```diff
@@ -0,0 +1,124 @@
+part of '../../main.dart';
+
+class PrivateViewPage extends StatefulWidget {
+  const PrivateViewPage({Key? key}) : super(key: key);
+
+  @override
+  _PrivateViewPageState createState() => _PrivateViewPageState();
+}
+
+class _PrivateViewPageState extends State<PrivateViewPage> {
+  late VideoPlayerController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+    _controller = VideoPlayerController.networkUrl(
+      Uri.parse(
+          'https://flutter.github.io/assets-for-api-docs/assets/videos/bee.mp4'),
+    )..initialize().then((_) {
+        setState(() {});
+      });
+  }
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Page(title: 'Private views', children: [
+      SingleChildScrollView(
+        child: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              const SizedBox(height: 16),
+              InstabugPrivateView(
+                child: const Text(
+                  'Private TextView',
+                  style: TextStyle(fontSize: 18),
+                  textAlign: TextAlign.center,
+                ),
+              ),
+              const SizedBox(height: 16),
+              InstabugPrivateView(
+                child: ElevatedButton(
+                  onPressed: () {
+                    // Navigate to a private web view
```

Comment:
@ahmedAlaaInstabug, Did you forgot to add action here ??

Reply:
@AndrewAminInstabug  fixed